<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWindow</class>
 <widget class="QMainWindow" name="ConnectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>401</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <widget class="QLineEdit" name="userText">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>60</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Пользователь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>430</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#connectButton{
background-color:rgba(255, 255, 255, 255);
}
QPushButton#connectButton:pressed{
padding-left:5px;
padding-right:5px;
background-color:rgba(0, 0, 0, 100);
background-position:calc(100%-10px)center;
}
QPushButton#connectButton{
background-color:rgba(255, 255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>П О Д К Л Ю Ч И Т Ь С Я</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordText">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>150</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="databaseText">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>240</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Имя базы данных</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostText">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>330</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Хост</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
